proto/sge/bet/constraints.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+package sgenetwork.sge.bet;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/sge-network/sge/x/bet/types";
+
+// PlacementConstraints is the bet constrains type for the bets
+message PlacementConstraints {
```

Comment:
BetConstraints would be better description

Reply:
Done